rel/files/gestalt.cmd

```diff
@@ -0,0 +1,103 @@
+@setlocal
```

Comment:
He did say, Mr. @rexim, that this code is generated. I still haven't
reviewed it, but I think that issue is not so valuable for us to mess with
this generated code.